wallettool/src/main/java/org/bitcoinj/wallettool/WalletTool.java

```diff
@@ -268,21 +268,21 @@ public Condition(String from) {
             else if (from.startsWith(">")) type = Type.GT;
             else throw new RuntimeException("Unknown operator in condition: " + from);
 
-            String s;
+            String ComparisonString;
```

Comment:
I think this could/should be simplified even further by eliminating this local variable altogether and just assigning `value` directly in the two `case` statements. If you want to submit that change as a separate PR, I would approve it. (But @schildbach 's approval is required for a merge.)

Also, standard Java coding conventions require that local variable start with a lowercase letter, so this would be `comparisonString`. 

This would also be a great candidate for a [switch expression](https://stackoverflow.com/questions/65657169/what-are-switch-expressions-and-how-are-they-different-from-switch-statements) at some time in the future when we might be using JDK 14 or later.